bin/uwsgi.ini

```diff
@@ -1,8 +1,32 @@
 [uwsgi]
-module = basket.wsgi:application
-http = :8000
 master = true
-vacuum = true
-enable-threads = true
-lazy-apps = true
+strict = true  ; Fail on unknown config options
+
+http = :8000
+module = basket.wsgi:application
+
+die-on-term = true  ; Shutdown on SIGTERM (default is respawn)
+enable-threads = true  ; Support multithreading
+harakiri = 20  ; Kill hung requests after 20 seconds
```

Comment:
I think gunicorn's default is 30secs. I doubt we'll notice but worth keeping an eye on if we get more timeouts